### ISSUE TITLE
Blacklist for Specific Sites

### ISSUE THREAD

**Issue by Kippenhof** [ajayyy/SponsorBlock#730]
Hey There,

I love that Addon so far, but i also love to use the Site twoseven.xyz.
The Problem is, that it skips the Ad, but only Locally. (probably because the Way Twoseven works)

It would be nice, if you could just add Twoseven on a Blacklist, or if it will skip the ADs for all participants.

**Comment by wilkmaciej** [comment]
Skipping ad's for everyone is out of question, SponsorBlock can only skip ads if it's installed in the browser on the person that is watching. Either tell your friends to install it (not sure if it will synchronize the video well) or disable it from extension popup before starting video on twoseven. It is possible to prevent skipping but hard coding sites on which SponsorBlock should not skip segments seems to me like not a good idea. If we wanted to add blacklist probably a better option would be to allow users to add sites to it from the popup UI.

**Comment by Kippenhof** [comment]
thats sad. I try to disable it every time when i enter the Twoseven Site, but sometimes i forget to deactivate it, and so the vid isn't synchronized anymore and i don't notice. Idk how hard something like a button is, that deactivates SponsorBlock if you are on a 3. Party Site. Normally i Watch Youtube only on the Official Page, so it would be great to disable it on 3. Party Sites.

**Comment by mchangrh** [comment]
duplicate of #554
